chart/values.yaml

```diff
@@ -0,0 +1,22 @@
+secretName: svc-questionnaires-secret
+gitlabAuthSecretName: gitlab-auth
+dockerAuthSecretName: docker-auth
```

Comment:
I have 1 last comment about this:

currently we are required to provide two secrets for image pull credentials: gitlab-auth and docker-auth. This is actually a hack because we were migrating from gitlab to docker. Now, when the migration is done we will forever be required to provide gitlab secrets because of this.

I suggest you create a list in values.yaml with imagePullSecretNames, and iterate over these secrets wherever they are used. That way, when we have migrated (which we already have) we don't need two secrets anymore.